Dockerfile

```diff
@@ -1,56 +1,32 @@
-# syntax=docker/dockerfile:1.2
+FROM bitnami/minideb:bullseye AS setup
 
-# build base that sets up common dependencies of the build
-FROM rust:alpine as build-base
+RUN apt-get update && apt-get install -y \
+  curl
+  #libgcc
+  # libstdc++
 
-RUN apk add --no-cache \
-  clang clang-dev clang-libs pkgconfig bearssl-dev git \
-  gcc make g++ linux-headers protobuf protobuf-dev musl-dev
+RUN mkdir /tmp/vitalam-node/
```

Comment:
I don't think you need a `mkdir` if you're then going to to a `WORKDIR`. It'll happen automatically I think